test/integration/simple_example/controls/server.rb

```diff
@@ -324,8 +324,6 @@
             including(
               "resource" => "iam_policy_violations",
               "should_notify" => true,
-              # This extra bit of verification tests the `iam_policy_violations_slack_webhook` variable
```

Comment:
@bmenasha Perhaps we could repurpose this test? I'm fine with dropping it though.

Reply:
I'm thinking about moving the whole template into it's own sub-module in the hopes of writing more tests against it. but i don't want to do that in this pull request.

Reply:
Got it, feel free to follow up.